_posts/2018/2018-04-03-what-a-code-of-conduct-means.md

```diff
@@ -0,0 +1,112 @@
+
+This article lays out the process and principles that I currently use to maintain the safety of my attendees and encourage the best behaviour of everyone involved.
+
+# On fairness, intent and privilege
+
+I am a British, white, middle-class and (increasingly) middle-aged man. This means I cannot necessarily accurately understand the experiences of people who do not share my in-built societal privilege.
+
+I do my best as a ‘very visible for one day’ member of our community to empathise and ‘do right’ by individuals form groups who aren’t represented in our community in the same numbers as people who look like me.
+
+This means that I will trust individuals to accurately report their understanding of other’s behaviour as it effects them. And I will err on the side of trusting the opinion of those offended, rather than trying to judge solely from my perspective.
+
+It feels hard to exclude people from an event, for ‘just saying things’ or for ‘being a bit creepy’, but I say that _from my perspective of cultural safety_.
+
+As for individuals who may feel harshly treated for behaviour they perceived as fine... the intent of your behaviour doesn’t matter. The effect of your behaviour matters. The perceived ‘correctness’ of your argument doesn't matter.
+
+Censure of your behaviour may not feel ‘fair’ but from the broader perspective of an event organiser I would rather err on the side of understanding and empathising with anyone made to feel unsafe by another attendee.
+
+In the case of small conversational infractions, or where you’ve expressed yourself awkwardly I’d suggest the following. A genuine apology (avoid qualifying phrases like ‘for causing offence’) and a resolution to talk less, smile more and think about how better to express yourself next time.
+
+
+## In the case of an incident at Brighton Ruby, reported during the conference
+
+I will err on the side of the victim/reporter.
+
+If I receive notice of a contravention of the behavioural standards expected of attendees I will ascertain the detail of the issue as quickly as possible, including any third parties to the incident.
+
+In the first instance I will seek the opinion of the victim/reporter as to the preferred course of action. This includes immediate expulsion of the perpetrator if it will make the victim feel safer in the Brighton Ruby environment. It’s important to say this is an option that is very much on the table, as I take the Code of Conduct very seriously.
+
+If the victim doesn’t want the individual to leave the conference I will arrange any resolution required by the individual. This might include a chaperone, or some form of safer space for the individual.
```

Comment:
Could maybe do with a comma after "the conference".
And you use the word "individual" to represent two people in this sentence.
Maybe replace "required by" with "agreed with"?